tests/gp/test_hsgp_approx.py

```diff
@@ -178,7 +178,7 @@ def test_prior(self, model, cov_func, X1, parameterization):
             gp = pm.gp.Latent(cov_func=cov_func)
             f2 = gp.prior("f2", X=X1)
 
-            idata = pm.sample_prior_predictive(samples=1000)
+            idata = pm.sample_prior_predictive(samples=1000, random_seed=rng)
```

Comment:
🙏

Reply:
One day we will get this 5 line PR merged. Just you wait :D